### PR TITLE
Fix for Left Menu isn't visible

### PR DESCRIPTION
**Pull request by prashant-80**
Screenshot 2024-10-01 at 1 14 28 PM
Screenshot 2024-10-01 at 1 14 03 PM

### CODE REVIEW

File: frontend/src/components/LeftMenu/LeftMenu.css [hunk 1]
```diff
@@ -7,6 +7,7 @@
   box-shadow: 0px 4px 6px rgba(81, 81, 81, 0.1);
   border-right: 1px solid #EBEBEB;
   min-width: max-content;
+  height: 100vh;
```
[the remark this answers is not in the document]
Comment (prashant-80) [reply frontend/src/components/LeftMenu/LeftMenu.css]: fixed it

**Comment by erenfn** [comment]
@gorkem-bwl 

image

I think there is too much space between the profile and the buttons

**Comment by prashant-80** [comment]
I adjusted the pixels can you check if it's fine?
Screenshot 2024-10-03 at 5 07 26 PM
Screenshot 2024-10-03 at 4 40 18 PM

File: frontend/src/components/LeftMenu/LeftMenu.css [hunk 2]
```diff
@@ -50,3 +50,57 @@
...
+    font-size: 10px;
+    line-height: 32px;
+  }
+
+  .left-menu .MuiListItemButton-root {
+    padding-top: 2px;
+    padding-bottom: 2px;
+  }
+
+  .search-item {
```
Comment (erenfn) [inline frontend/src/components/LeftMenu/LeftMenu.css:91]: Pull the latest code from develop. We don't have search-item class anymore.

File: frontend/src/components/LeftMenu/LeftMenu.css [hunk 3]
```diff
@@ -50,3 +50,57 @@
...
+
+  .left-menu .MuiListItemButton-root {
+    padding-top: 4px;
+    padding-bottom: 4px;
+  }
+}
+
+@media screen and (max-height: 615px) {
+  .menu-item .MuiTypography-root {
+    font-size: 12px;
```
Comment (erenfn) [inline frontend/src/components/LeftMenu/LeftMenu.css:77]: Don't forget to resize the icons too.

**Review by erenfn — CHANGES_REQUESTED** [review]

**Review by gorkem-bwl — APPROVED** [review]
Looks better!

File: frontend/src/components/LeftMenu/LeftMenu.css (hunk 3 above)
Comment (prashant-80) [reply frontend/src/components/LeftMenu/LeftMenu.css:82]: I adjusted it

File: frontend/src/components/LeftMenu/LeftMenu.css (hunk 2 above)
Comment (prashant-80) [reply frontend/src/components/LeftMenu/LeftMenu.css:96]: In develop branch it has search-item class

**Review by erenfn — CHANGES_REQUESTED** [review]
It is good to go after you delete 

- [ ] .search-item in LeftMenu.css line 30 
- [ ] Delete .DS_Strore

**Review by erenfn — APPROVED** [review]

**Review by erenfn — APPROVED** [review]

**Pull request merged**